hypertrace-graphql-labels-schema/src/main/java/org/hypertrace/graphql/label/schema/mutation/LabelMutationSchema.java

```diff
@@ -22,5 +22,5 @@
   @GraphQLNonNull
   @GraphQLName(UPDATE_LABEL)
   @GraphQLDataFetcher(LabelUpdateMutator.class)
-  Label updateLabel(@GraphQLNonNull @GraphQLName(Label.ARGUMENT_NAME) Label label);
+  Label updateLabel(@GraphQLNonNull @GraphQLName(UpdateLabel.ARGUMENT_NAME) UpdateLabel label);
```

Comment:
nit: match code arg names to declared arg names
```suggestion
  Label updateLabel(@GraphQLNonNull @GraphQLName(UpdateLabel.ARGUMENT_NAME) UpdateLabel updateLabel);
```

(same for create)